tests/models/test_lora_layers.py

```diff
@@ -537,3 +536,34 @@ def test_vanilla_funetuning(self):
         expected = np.array([0.7406, 0.699, 0.5963, 0.7493, 0.7045, 0.6096, 0.6886, 0.6388, 0.583])
 
         self.assertTrue(np.allclose(images, expected, atol=1e-4))
+
+    def test_unload_lora(self):
```

Comment:
Contingent on: https://github.com/huggingface/diffusers/issues/4055

Reply:
Nice! Could we maybe also add a fast test as it'll be important to make sure this method stays correct